### ISSUE TITLE
Bigboard may break with kanboard 1.2.3

### ISSUE THREAD

**Issue by kgeil** [stinnux/kanboard-bigboard#7]
My apologies, this is a big edit from the first one an hour or two ago.  It's in an effort to not cross-post with the issues on kanboard.

My organization is beginning to utilize kanboard, and several of us love the "Bigboard" plugin, developed by @stinnux .  I updated from version 1.2.1 to version 1.2.3 today.

### Actual behaviour
When I click on "projects management>Bigboard, I only see one project at the top (with all tasks from that first project), and underneath that, instead of seeing the next project in its entirety, I only see the title, without displaying tasks below.  I reverted to the previous version, and Bigboard works again.

### Expected behaviour: 

When I click on the Bigboard plugin, all projects for which I have permission should show up, one beneath the other.

  I hope I find the time to look into this myself.  Thanks to @fguillot and other contributors for such a great project!  #Attached are debug logs from kb 1.2.3 and 1.2.1.  There shouldn't be much in them except for authentication, then navigation to projects management, then bigboard.  I'll gather more information asap.

### Logs

[KB123debug.log](https://github.com/kanboard/kanboard/files/1961995/KB123debug.log)
[KB121debug.log](https://github.com/kanboard/kanboard/files/1961997/KB121debug.log)

r more information tomorrow.

Thank you.

Kevin T. Geil

- [X] Bug report
- [ ] Feature request
- [ ] Improvement

- Please, read the [Frequently Asked Questions](https://docs.kanboard.org/en/latest/developer_guide/faq.html).
- Ask questions on the [Forum](https://github.com/kanboard/forum/issues).

### Unexpected behavior (can't see projects under Implementation)

![image](https://user-images.githubusercontent.com/10849557/39451736-ae94b24a-4c9d-11e8-9150-be24daae4e7d.png)

### Expected behavior on 1.2.1:  (all projects are visible).

![image](https://user-images.githubusercontent.com/10849557/39451902-4af5c7e6-4c9e-11e8-8107-649cb4329483.png)

### Configuration

- Kanboard version: 1.2.3
- Database type and version: 
- PHP version:  PHP 7.0.28-0
- OS: ubuntu0.16.04.1
- Browser: Chrome, Firefox, IE

**Comment by stinnux** [comment]
I am unsure, and can't find it in the logs. Did you also upgrade the Plugin? Or only kanboard?

**Comment by kgeil** [comment]
Thanks.  I did upgrade the plugin as well.  I'll try to get some better logs to you a bit later today.

Kevin

**Comment by kgeil** [comment]
Thanks for looking, @stinnux. The plugin version is 1.0.2.  I did install kanboard 1.2.2 on my test server.  When I upgraded to kanboard 1.2.3, it failed as described above.
Below are 2 logs:  Kanboard121.log refers to kanboard version 1.2.1 Kanboard 123.log refers to kanboard version 1.2.3.  

[Kanboard121.log](https://github.com/stinnux/kanboard-bigboard/files/1964352/Kanboard121.log)
[Kanboard123.log](https://github.com/stinnux/kanboard-bigboard/files/1964355/Kanboard123.log)
